docs/source/common_issues.rst

```diff
@@ -582,7 +582,7 @@ the same line as the import:
 
    # to silence complaints about unused imports
    from typing import List  # noqa
-   a = None  # type: List[int]
+   a = []  # type: List[int]
```

Comment:
In the original example, `mypy` reports `error: Incompatible types in assignment (expression has type "None", variable has type "List[int]")`.